### PR TITLE
Fix versioning using setuptools_scm and leave versioneer.py

### PR DESCRIPTION
**Pull request by adybbroe**
Replace versioneer.py using setuptools_scm instead. versioneer is not maintained.

 - [ ] Closes #xxxx 
 - [ ] Tests added  
 - [ ] Tests passed 
 - [ ] Passes ``flake8 trollflow2`` 
 - [ ] Fully documented 
 - [x] Add your name to `AUTHORS.md` if not there already

### CODE REVIEW

**Comment by adybbroe** [comment]
I need this to continue fixing the doc-pages...

File: doc/source/conf.py [hunk 1]
```diff
@@ -20,15 +20,22 @@
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
 
+from pkg_resources import get_distribution
+
+# The version info for the project you're documenting, acts as replacement for
+# |version| and |release|, also used in various other places throughout the
+# built documents.
+#
+release = get_distribution('satpy').version
```
Comment (pnuu) [inline doc/source/conf.py]: Copy&paste error: `satpy` -> `trollflow2`

Comment (mraspaud) [reply doc/source/conf.py]:
```suggestion
release = get_distribution('trollflow2').version
```

Comment (mraspaud) [reply doc/source/conf.py]: @adybbroe could you commit the suggestion above, so we can merge this ?

**Pull request merged**